README.md

```diff
@@ -46,14 +49,32 @@ Supported Golang version:
 
 This package can be installed with the go get command:
 
-    go get github.com/mattn/go-sqlite3
+    go get github.com/CovenantSQL/go-sqlite3-encrypt
```

Comment:
ditto

Reply:
I changed that just in case you dont want to merge my pr.
so, any other modify I need to do?